packages/fast-components-react-base/src/image/image.tsx

```diff
@@ -5,101 +5,69 @@ import Foundation, { HandledProps } from "../foundation";
 import { IImageHandledProps, IImageManagedClasses, IImageUnhandledProps } from "./image.props";
 import { IImageClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
 
-/**
- * TODO #239 #240: Hook viewports values into Styles
- */
-export interface IBreakpoints {
-    vp1: number;
-    vp2: number;
-    vp3: number;
-    vp4: number;
-    vp5: number;
-    vp6: number;
+export enum ImageSlot {
+    source = "source"
 }
 
-const breakpoints: IBreakpoints = {
-    vp1: 0,
-    vp2: 540,
-    vp3: 768,
-    vp4: 1084,
-    vp5: 1400,
-    vp6: 1779
-};
-
 /* tslint:disable-next-line */
 class Image extends Foundation<IImageHandledProps & IManagedClasses<IImageClassNameContract>, React.HTMLAttributes<HTMLImageElement | HTMLPictureElement>, {}> {
     protected handledProps: HandledProps<IImageHandledProps & IManagedClasses<IImageClassNameContract>> = {
         managedClasses: void 0,
         alt: void(0),
-        itemScope: void(0),
         sizes: void(0),
         src: void(0),
-        srcSet: void(0),
-        vp1: void(0),
-        vp2: void(0),
-        vp3: void(0),
-        vp4: void(0),
-        vp5: void(0),
-        vp6: void(0)
+        srcSet: void(0)
     };
 
     /**
      * Renders the component
      */
     public render(): React.ReactElement<HTMLImageElement | HTMLPictureElement> {
-        if (!this.props.src && !this.props.vp1) {
-            return;
-        }
+        let className: string = get(this.props, "managedClasses.image");
 
-        if (this.props.src) {
+        if (!this.props.children) {
             return (
                 <img
                     {...this.unhandledProps()}
-                    className={super.generateClassNames(get(this.props, "managedClasses.image"))}
+                    className={super.generateClassNames(className)}
                     alt={this.props.alt}
                     sizes={this.props.sizes ? this.props.sizes : null}
```

Comment:
I wonder if we want to do something like `{...this.props. sizes && {'sizes' : this.props.sizes}}` so it doesn't get written to the DOM if it doesn't exist

Reply:
This doesn't get written to the DOM if it doesn't exist.

Reply:
I guess I was seeing it in the snapshots

Reply:
That's due to our jest version. they pull all null || undefined types in a later version, but we have not updated.